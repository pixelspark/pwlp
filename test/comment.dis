0000.	11	PUSHB	[05]
0002.	fe	SPECIAL	yield
0003.	fd	SPECIAL	dump
0004.	71	UNARY	DEC
0005.	60	JNZ	to 2
0008.	01	POP	1

