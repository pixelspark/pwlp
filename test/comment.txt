/* x */ for(i=5) {
	/* xx */ /* xx */
	yield /* x */;
	/* xx */ /* xx */
	// single line comment
	dump;
}/*x */