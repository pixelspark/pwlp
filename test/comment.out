DUMP: [5]
DUMP: [4]
DUMP: [3]
DUMP: [2]
DUMP: [1]
